FirstProject/Colours.cs

```diff
@@ -0,0 +1,33 @@
+﻿
+using System;
+
+namespace FirstProject
+{
+    internal class Colours
+    {
+        public void WriteLineGreen(string text, ConsoleColor color)
```

Comment:
Po co ten drugi parametr? Ogólnie nwm czy nie lepiej by w tej klasie zrobić jedną klasę której przezywalismybyśmy parametr ConsoleColor zamiast robic 4 metody których różnice to tylko jedna linia